LUFA/Drivers/USB/Class/Common/HIDClassCommon.h

```diff
@@ -330,6 +330,23 @@
 		#define HID_KEYBOARD_SC_RIGHT_SHIFT                       0xE5
 		#define HID_KEYBOARD_SC_RIGHT_ALT                         0xE6
 		#define HID_KEYBOARD_SC_RIGHT_GUI                         0xE7
+		#define HID_KEYBOARD_SC_MEDIA_PLAY                        0xE8
```

Comment:
These usages aren't listed in the keyboard/keypad page. Are you sure this is the right place for them?

Reply:
I just stumbled across that code wondering why those definitions are there. They are wrong and do not belong to this file. The media keys have different values. If you declare those definitions "Scancodes - SC" it is wrong. We should revert this change.

Reply:
Admittedly, the consumer page should be used for media key codes, but they aren't completely wrong. Back in the days, I didn't know about the consumer page and only used the keyboard / keypad page. These values worked for me for Linux, but later I realized that they don't work for other OSes. Given that they work on Linux, I think they're not futile. Maybe it should be emphasized that they only work for Linux.

Reply:
If you use those scancodes under linux they work as media keys!?!? They re not even documented. I need to test it, that would be wonderful, but still weird.

Reply:
Yes, the last time they did work on Linux. They're not documented, indeed.

Reply:
It seems to work. I am really surprised. This is ultra awesome!